ui/src/main/javascript/app/login/login.tpl.html

```diff
@@ -1,17 +1,15 @@
-<div class="top-image">
-    <img class="image-title" ng-src="resources/images/logo.png" />
-</div>
-<div class="sw-box" name="form" novalidate>
-    <div class="forceColor"></div>
-    <div class="topbar">
-        <div class="spanColor"></div>
-        <input name="user" type="text" class="sw-input" placeholder="Name" ng-model="user" required/>
-
-        <button class="sw-button" id="submit" type="submit" ng-disabled="form.user.$invalid" ng-click="$ctrl.doLogin(user)">Login</button>
+<div class="login">
+    <div class="top-image">
+        <img class="image-title" ng-src="resources/images/logo.png" />
+    </div>
+    <div class="form">
+        <div class="sw-box" name="form" novalidate>
+            <input name="user" type="text" class="sw-input" placeholder="Name" ng-model="user" required/>
```

Comment:
Change for: 
<input class="sw-input" ng-model="user" name="user" type="text" placeholder="Name" required/>
Class attribute is previous to standar attributes